TestApp/ios/Pods/AppCenter/AppCenter-SDK-Apple/iOS/AppCenterData.framework/Headers/AppCenterData.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
```

Comment:
I suppose this one can be omitted

Reply:
This is the pods being committed, its not react native repo code.